sagansystems/raven-go
# Upgrade to upstream master

Merge in the latest changes from getsentry/raven-go

**why**

We were using the new style SENTRY_DSN in the environment, which was causing sentry not to report supernova issues on master.

![](http://take.ms/uY0hK)

**context**

https://sagan.slack.com/archives/C0NTGR265/p1528146998000519

**testing**

1. Merge this into supernova. 
2. Set the `SENTRY_DSN` environment variable to the new style DSN.
3. Hit http://localhost:8001/error
4. Ensure the resulting error is reported to sentry.
